sdks/python/apache_beam/io/mongodbio_it_test.py

```diff
@@ -21,13 +21,12 @@
 import logging
 import time
 
-from pymongo import MongoClient
```

Comment:
Note that I had to fix the mongodb imports due to pylint failing for me even though it passes at head.